src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -104,7 +104,7 @@ const useSaveAnonChatHook = () => {
       console.log(e);
     } finally {
       refresh('channelList');
-      refresh('chat');
+      refreshWithId('chat', initAnonymousChat.message.cid);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refreshWithId` function is being called with 'chat' and `initAnonymousChat.message.cid` as parameters. Make sure that `initAnonymousChat.message.cid` always contains a valid value and handle cases where it might be null or undefined to prevent runtime errors.
```diff
-  refresh('chat');
+  refreshWithId('chat', initAnonymousChat.message.cid);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->